packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## v7.0.0-dev.0 (May 20, 2025)
 
 - Improved the layout of product tiles in product scroll and product list [#2446](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2446)
+- TODO
```

Comment:
?

Reply:
I added a todo there to bypass the changelog check. But I'll remove it later.